packages/editor/src/store/actions.js

```diff
@@ -26,22 +26,58 @@ import {
 } from './utils/notice-builder';
 
 /**
- * Returns an action object used in signalling that editor has initialized with
+ * WordPress dependencies
+ */
+import {
+	parse,
+	synchronizeBlocksWithTemplate,
+} from '@wordpress/blocks';
+
+/**
+ * Returns an action generator used in signalling that editor has initialized with
  * the specified post object and editor settings.
  *
  * @param {Object} post      Post object.
  * @param {Object} edits     Initial edited attributes object.
  * @param {Array?} template  Block Template.
- *
- * @return {Object} Action object.
```

Comment:
Nit: I'd wondered if JSDoc had some option to document generator function yields. It appears `@yields` is a thing, a suitable replacement for `@return`.

http://usejsdoc.org/tags-yields.html

Reply:
Ya I had thought about using `@yields` but the difficulty here is that most if not all of these actions do not have a single yield.  So it's a bit tricky knowing what exactly to document.

I've sort of loosely followed the pattern of only include a `@return` tag on generators when there are return values in the generator, otherwise having nothing.

Reply:
Added `@yields` as a feature request for the `docgen` default formatter: https://github.com/WordPress/gutenberg/issues/14583